### ISSUE TITLE
Q: is `createSVGMatrix` deprecated Web API used on purpose?

### ISSUE THREAD

**Issue by ankostis** [ariutta/svg-pan-zoom#373]
According to [the SVG2 standard](https://developer.mozilla.org/en-US/docs/Web/API/SVGMatrix), 
`createSVGMatrix()` is has been replaced with [`https://developer.mozilla.org/en-US/docs/Web/API/DOMMatrix`](https://developer.mozilla.org/en-US/docs/Web/API/DOMMatrix).

Is there any plan to replace the use of this API?

**Comment by ariutta** [comment]
No, there's no special reason other than my job has shifted focus away from front-end. Any chance you'd be interested in becoming a maintainer?

**Comment by ankostis** [comment]
No, sorry, I'm in the same position as you are :-)
